ipam/allocator.go

```diff
@@ -46,9 +77,11 @@ type Allocator struct {
 // NewAllocator returns an instance of libnetwork ipam
 func NewAllocator(ds datastore.DataStore) (*Allocator, error) {
 	a := &Allocator{}
-	a.subnets = make(map[subnetKey]*SubnetInfo)
+	a.subnets = make(map[subnetKey]*PoolData)
 	a.addresses = make(map[subnetKey]*bitseq.Handle)
-	a.internalHostSize = defaultInternalHostSize
+	a.predefined = make(map[string][]*net.IPNet, 2)
+	a.predefined[localAddressSpace] = localPredefinedPools
+	a.predefined[globalAddressSpace] = globalPredefinedPools
 	a.store = ds
 	a.App = "ipam"
 	a.ID = dsConfigKey
```

Comment:
From what I understand the `KeyPrefix` for allocator is made of only `a.App` and `a.ID`. Where is `addressSpace` in the `KeyPrefix`?